queries.sql

```diff
@@ -6,3 +6,42 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals
+ROLLBACK;
```

Comment:
- [ ] Again here we have to verify that the change was made using select statements e.g. `SELECT COUNT(*) FROM ANIMALS;`, before and after the `ROLLBACK` statement, kindly fix this.